iOS/Plugins/DependencyPlugin/ProjectDescriptionHelpers/Dependency+Target.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Dependency+Target.swift
+//  IOS
+//
+//  Created by 홍승현 on 11/19/23.
+//
+
+import ProjectDescription
+
+// MARK: - Feature
+
+public enum Feature: String {
+  case record
+
+  public var targetName: String {
+    return rawValue.prefix(1).capitalized + rawValue.dropFirst()
+  }
+}
+
+public extension TargetDependency {
+  static let designSystem: TargetDependency = .project(target: "DesignSystem", path: .relativeToShared("DesignSystem"))
+  static let trinet: TargetDependency = .project(target: "Trinet", path: .relativeToCore("Network"))
+  static let coordinator: TargetDependency = .project(target: "Coordinator", path: .relativeToCore("Coordinator"))
+  static let combineCocoa: TargetDependency = .project(target: "CombineCocoa", path: .relativeToShared("CombineCocoa"))
+
```

Comment:
P1

저희 TNCombnineCocoa로 알고 있습니다!
```suggestion
  static let combineCocoa: TargetDependency = .project(target: "TNCombineCocoa", path: .relativeToShared("TNCombineCocoa"))

```